libs/langchain-community/package.json

```diff
@@ -35,6 +35,7 @@
   "author": "LangChain",
   "license": "MIT",
   "dependencies": {
+    "@hashgraph/sdk": "^2.53.0",
```

Comment:
Should be a peer + dev dep, not a direct dependency:

https://github.com/langchain-ai/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md#third-party-dependencies

Reply:
Will be done by EOD Monday. Are there any other changes we need to make?